src/main/java/com/github/timm/cucumber/generate/CucumberITGenerator.java

```diff
@@ -188,9 +186,14 @@ private String createFormatStrings() {
 
         for (int i = 0; i < formatStrs.length; i++) {
             final String formatStr = formatStrs[i].trim();
-            sb.append(String.format("\"%s:%s/%s.%s\"", formatStr,
-                            config.getCucumberOutputDir().replace('\\', '/'), fileCounter,
-                            formatStr));
+
+            if ("pretty".equalsIgnoreCase(formatStr)) {
+                sb.append("\"pretty\"");
```

Comment:
hmm, is there a list of plugins where the `:PATH` part is not desirable (or vice versa)?

I think it would be better to treat these as a group - plugins either require a path or they don't,

Which would make the above line:

`sb.append("\"%s\"", formatStr);`

Special cases make me worried :)